### ISSUE TITLE
main-cron: meta store restore test failed

### ISSUE THREAD

**Issue by BugenZhao** [risingwavelabs/risingwave#14672]
```
Other(
--
  | "snapshot id 4 not found",
  | )
  | thread 'main' panicked at src/cmd/src/lib.rs:78:10:
  | called `Result::unwrap()` on an `Err` value: snapshot id 4 not found
  |  
  | Stack backtrace:
  | 0: msg<alloc::string::String>
  | at ./.cargo/registry/src/index.crates.io-6f17d22bba15001f/anyhow-1.0.75/src/error.rs:83:36
  | 1: {async_fn#0}
  | at ./src/meta/src/backup_restore/restore.rs:124:39
  | 2: {async_fn#0}
  | at ./src/meta/src/backup_restore/restore.rs:190:49
  | 3: {async_fn#0}
  | at ./src/ctl/src/lib.rs:694:60
  | 4: {async_fn#0}
  | at ./src/ctl/src/lib.rs:530:45
```

https://buildkite.com/risingwavelabs/main-cron/builds/1514#018cfe67-8013-49d1-91b9-d76b21fc1891

cc @zwang28 Would you please help to take a look?

**Comment by zwang28** [comment]
It has already been fixed by #14567

**Comment by BugenZhao** [comment]
Oops, I didn't find that. Thanks for the fix.
